### ISSUE TITLE
Delete all points

### ISSUE THREAD

**Issue by condairteam** [ichim/LeafletForBlazor-NuGet#43]
Hi Sir,

Thank you for the incredible update, working everything is just fine with the last update (2.0.8.4) ,  but one function, which is the delete function. 
When i am calling  this:
```c#
 await _realTimeMap.Geometric.Points.delete(); 
```
i got this error:
```
 System.InvalidOperationException: Sequence contains no elements
         at System.Linq.ThrowHelper.ThrowNoElementsException()
         at System.Linq.Enumerable.MaxFloat[TSource,TResult](IEnumerable`1 source, Func`2 selector)
         at LeafletForBlazor.RealTime.points.Points.updateExtent()
         at LeafletForBlazor.RealTime.points.Points.delete()
```
No matter there are points on the map (which are just showing fine) or just an empty map.

Thanks,
Alex

**Comment by ichim** [comment]
I am sorry.
I will fix it

**Comment by ichim** [comment]
....in a few minutes version 2.0.8.6 will be available. This version fixes bug #43 reported by you

**Comment by condairteam** [comment]
Nice working fine again. Good job 💯 👍
